tests/neg/i2887b.scala

```diff
@@ -0,0 +1,13 @@
+trait A { type S[X[_] <: [_] => Any, Y[_]] <: [_] => Any; type I[_] } // error // error
```

Comment:
There are errors in this file. The syntax of type lambdas is now `[_] =>>` rather than `[_] =>`.